mpeg2/src/interleaving_muxer.rs

```diff
@@ -596,4 +598,38 @@ mod test {
             ]
         );
     }
+
+    #[test]
+    fn test_audio_video_pts_from_wall_clock() {
```

Comment:
Naming suggestion since it looks like the crux of the issue is that the input presentation times were higher than expected, and not necessarily that they were derived from wall clock times:

```suggestion
    fn test_oversized_presentation_times() {
```